app/controllers/single_page_subscriptions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class SinglePageSubscriptionsController < ApplicationController
+  include AccountHelper
+  include GovukPersonalisation::ControllerConcern
+
+  before_action do
+    head :not_found unless govuk_account_auth_enabled?
+  end
+
+  def show
+    content_item = GdsApi.content_store.content_item(topic).to_h
+    if logged_in?
```

Comment:
You could avoid the big indented block by changing this line to `return unless logged_in?`